source/jobs.c

```diff
@@ -346,9 +316,9 @@ JobsStatus_t Jobs_GetTopic( char * buffer,
  * @return JobsSuccess if the sequences are the same;
  * JobsNoMatch otherwise
  */
-static JobsStatus_t strnEq( const char * a,
-                            const char * b,
-                            size_t n )
+static JobsStatus_t strnEquals( const char * a,
```

Comment:
Curious why was this renamed.

Reply:
coreJSON also has a static strnEq() function which was a violation, I forget the exact rule number. Speaking with @kstribrnAmzn, renaming this to be the full name "strnEquals" seemed to be the most straight-forward solution.